### DISCUSSION TITLE
0.6.2 - Minor updates

### DISCUSSION THREAD

**Discussion by Timthreetwelve** [Timthreetwelve/MyScheduledTasks#19] [Announcements]
# The update features visual enhancements, introduces an alternative method to check for updates, and includes numerous dependency updates.

### Changes: 👀

- New: Check for updates from the three-dot menu
- Updated: Layout changes on About page
- Updated: Bump CommunityToolkit.Mvvm from 8.3.1 to 8.3.2
- Updated: Bump Vanara.PInvoke.User32 from 4.0.3 to 4.0.4
- Updated: Bump NLog from 5.3.3 to 5.3.4
- Updated: Bump Octokit from 12.0.0 to 13.0.1
- Updated: Bump MaterialDesignThemes from 5.0.0 to 5.1.0
- Removed: Removed GitVersion.MsBuild
- Added: NerdBank.GitVersioning

**Full Changelog**: <https://github.com/Timthreetwelve/MyScheduledTasks/compare/v0.6.1...v0.6.2>

**File Descriptions:**
| File Name | Description | Size |
| --- | --- | --- |
|MyScheduledTasks_0.6.2.0_x64x86_Setup.exe|Installer, **requires** .NET 8 runtime to already be present |8.83 MB|
|MyScheduledTasks_0.6.2.0_SC_x64_Setup.exe|Installer, x64 architecture, Self-Contained, .NET runtime **not required** |62.01 MB|
|MyScheduledTasks_0.6.2.0_SC_x86_Setup.exe|Installer, x86 architecture, Self-Contained, .NET runtime **not required** |57.24 MB|
|MyScheduledTasks_0.6.2.0_x64x86_Portable.zip|Zip archive, **requires** .NET 8 runtime to already be present |12.62 MB|
|MyScheduledTasks_0.6.2.0_SC_x64_Portable.zip|Zip archive, x64 architecture, Self-Contained, .NET runtime **not required** |85.65 MB|
|MyScheduledTasks_0.6.2.0_SC_x86_Portable.zip|Zip archive, x86 architecture, Self-Contained, .NET runtime **not required** |79.63 MB|

**Checksums:**
| File Name | SHA256 Hash | Virus Total |
| --- | --- | :---: |
|MyScheduledTasks_0.6.2.0_x64x86_Setup.exe|` d1d4b473d746df62c5a369fe1aa5abc3069928773d42cb159c5aab1f49f2cb40 `|[link](https://www.virustotal.com/gui/file/d1d4b473d746df62c5a369fe1aa5abc3069928773d42cb159c5aab1f49f2cb40) |
|MyScheduledTasks_0.6.2.0_SC_x64_Setup.exe|` 8171bc7eb30c143072ba39b90690cd57858f8077e676718ef3afbfb00d9c9567 `|[link](https://www.virustotal.com/gui/file/8171bc7eb30c143072ba39b90690cd57858f8077e676718ef3afbfb00d9c9567) |
|MyScheduledTasks_0.6.2.0_SC_x86_Setup.exe|` f4a2baec410d337cf89170b1f371cde92b3e6350a5c34acf756209c4174b5faa `|[link](https://www.virustotal.com/gui/file/f4a2baec410d337cf89170b1f371cde92b3e6350a5c34acf756209c4174b5faa) |
|MyScheduledTasks_0.6.2.0_x64x86_Portable.zip|` 77e72510350e884b7a4fcd9da00ce30702fb63f14e79fa4252eecef981e58274 `|[link](https://www.virustotal.com/gui/file/77e72510350e884b7a4fcd9da00ce30702fb63f14e79fa4252eecef981e58274) |
|MyScheduledTasks_0.6.2.0_SC_x64_Portable.zip|` b4b503460c3f7190c796fe3b9e74790ec5bafbc7ede833d4d0454d636a9fe23e `|[link](https://www.virustotal.com/gui/file/b4b503460c3f7190c796fe3b9e74790ec5bafbc7ede833d4d0454d636a9fe23e) |
|MyScheduledTasks_0.6.2.0_SC_x86_Portable.zip|` f9a295601461bcfe5c8049c6254887c34763b500f2b9e6c03050182d2d20c9af `|[link](https://www.virustotal.com/gui/file/f9a295601461bcfe5c8049c6254887c34763b500f2b9e6c03050182d2d20c9af) |

<hr /><em>This discussion was created from the release <a href='https://github.com/Timthreetwelve/MyScheduledTasks/releases/tag/v0.6.2'>0.6.2 - Minor updates</a>.</em>

**Comment by ghost** [comment]
Latest update in visual studio 22 , it is creating a issue . It is removing all the sdk and just adding .net 9 even if you have .net 8 or lower version . User need to download the other sdk if they want to work on other . I am not able to understand if .net 9  has short term support and .net 8 has long term support then why .net 8 is removed

**Comment by Timthreetwelve** [comment]
Hello @Vikas-Devnet 

I was also dismayed that .NET 8 was removed upon updating Visual Studio. What I did to fix the situation was to reinstall the .NET 8 SDK and add a `global.json` file that contains the following:

``` XML
{
  "sdk": {
    "version": "8.0.400"
  }
}
```
